automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/rstudio/RStudioSpec.scala

```diff
@@ -1,26 +1,26 @@
 package org.broadinstitute.dsde.workbench.leonardo.rstudio
 
 import org.broadinstitute.dsde.workbench.leonardo._
-import org.broadinstitute.dsde.workbench.leonardo.notebooks.NotebookTestUtils
+import org.openqa.selenium.{Keys}
 import org.scalatest.DoNotDiscover
-
-import scala.util.Try
-
 @DoNotDiscover
-class RStudioSpec extends RuntimeFixtureSpec with NotebookTestUtils {
+class RStudioSpec extends RuntimeFixtureSpec with RStudioTestUtils {
 
   override val toolDockerImage: Option[String] = Some(LeonardoConfig.Leonardo.rstudioBioconductorImage.imageUrl)
 
   "RStudioSpec" - {
 
     "should launch RStudio" in { runtimeFixture =>
-      withWebDriver { _ =>
-        // See this ticket for adding more comprehensive selenium tests for RStudio:
-        // https://broadworkbench.atlassian.net/browse/IA-697
-        val getResult = Try(RStudio.getApi(runtimeFixture.runtime.googleProject, runtimeFixture.runtime.clusterName))
-        getResult.isSuccess shouldBe true
-        getResult.get should include("unsupported_browser")
-        getResult.get should not include "ProxyException"
+      withWebDriver { implicit driver =>
+        withNewRStudio(runtimeFixture.runtime) { rstudioPage =>
+//          // See this ticket for adding more comprehensive selenium tests for RStudio:
+//          // https://broadworkbench.atlassian.net/browse/IA-697
+          Thread.sleep(10000)
+          rstudioPage.pressKeys("varA <- 1000")
+          rstudioPage.pressKeys(Keys.ENTER.toString)
+          Thread.sleep(10000)
```

Comment:
Curious if the sleeps are needed.. wonder if there is a way to use `Selenium` to `await` conditions instead of hard-coding sleeps.